dotgov-domains/current-federal.csv

```diff
@@ -896,6 +850,7 @@ ROBODEIDENTIDAD.GOV,Federal Agency,Federal Trade Commission,Washington,DC
 SENTINEL.GOV,Federal Agency,Federal Trade Commission,Washington,DC
 UCE.GOV,Federal Agency,Federal Trade Commission,Washington,DC
 18F.GOV,Federal Agency,General Services Administration,Washington,DC
+ACCESSIBILITY.GOV,Federal Agency,General Services Administration,Washington,DC

```

Comment:
Noting this for the record as a new domain, not just a metadata cleanup. I'm sure there are others here.